src/app/statement-notification/statement-notification.component.html

```diff
@@ -0,0 +1,7 @@
+<mat-card>
+  <mat-card-title>{{ statement.title }}</mat-card-title>
+  <mat-card-subtitle>New statement from the developers!</mat-card-subtitle>
```

Comment:
Make this part of the json to be customisable too. And this one should say "Statement from the project leads", I think.